config/manager/manager.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: system
```

Comment:
`name` should always be first under `metadata` IMO for readability/scan-ability. Silly generation code.